src/main/java/uk/nhs/hee/tis/trainee/forms/mapper/FormRPartAMapper.java

```diff
@@ -36,4 +37,8 @@
   List<FormRPartADto> toDtos(List<FormRPartA> formRPartAs);
```

Comment:
Is this method still used?

Reply:
It's not used atm, but I think these methods are just some basic mappers, and can be left for later use.